Docs/Learning_Resources.md

```diff
@@ -0,0 +1,71 @@
+# Learning Resources
```

Comment:
Is there a reason this is being put into a separate file? If we really think it is better to be in a separate file then we should have a link or something on the main README for people to notice it.

Reply:
Moving the learning resources to their own file is part of the new standard. I linked it in the main README under `Getting Help`